astro-run docs (#1532)
astronomer/docs: astro/develop-project.md

Once the project builds, you can access the Airflow UI by going to `http://localhost:8080/` and logging in with `admin` for both your username and password. You can also access your Postgres database at `localhost:5432/postgres`.

> You don't need to use the Airflow UI to run and debug your DAGs. You can use the `astro run` command to run and debug your DAGs from the command line. See [Run and Debug DAGs with Astro Run](test-and-troubleshoot-locally.md#run-and-debug-dags-with-astro-run).

Reviewer: ```suggestion
:::tip

Use the `astro run <dag-id>` command to run and debug a DAG from the command line without starting a local Airflow environment. This is an alternative to testing your entire Astro project with the Airflow webserver and scheduler. See [Run and Debug DAGs with Astro Run](test-and-troubleshoot-locally.md#run-and-debug-dags-with-astro-run).

:::
```

> You don't need to use the Airflow UI to run and debug your DAGs. You can use the `astro run` command to run and debug your DAGs from the command line. See [Run and Debug DAGs with Astro Run](test-and-troubleshoot-locally.md#run-and-debug-dags-with-astro-run).

Author: There is already and info and tip on this section. I thought adding another tip would look like too much

> You do not need to use the Airflow UI to run and debug your DAGs. You can use the `astro run` command to run and debug the DAGs within your Ariflow project with just the command line. See [Run and Debug DAGs with Astro Run](test-and-troubleshoot-locally.md#run-and-debug-dags-with-astro-run) for more information.

Reviewer 2: ```suggestion
You don't need to use the Airflow UI to run and debug your DAGs. You can use the `astro run` command to run and debug the DAGs within your Airflow project from the command line. See [Run and Debug DAGs with Astro Run](test-and-troubleshoot-locally.md#run-and-debug-dags-with-astro-run).
```
